rotkehlchen/errors/api.py

```diff
@@ -7,7 +7,7 @@ class RotkehlchenPermissionError(Exception):
     The payload contains information to be shown to the user by the frontend so
     they can decide what to do
     """
-    def __init__(self, error_message: str, payload: Optional[dict[str, Any]]) -> None:
+    def __init__(self, error_message: str, payload: Optional[dict[str, Any]] = None) -> None:
```

Comment:
Why do you need to add the default value here? Explicit is better than implicit. What does this have to do with the goal of your PR?